src/Server.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Server.hpp"
+#include "nlohmann/json.hpp"
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include "curl/curl.h"
+#include "Utility.hpp"
+#include "User.hpp"
+
+std::string data;
```

Comment:
shouldn't it be a member field in a Server class?

Reply:
Good point;) std::string data moved into class.